cmd/dagger/module.go

```diff
@@ -32,7 +32,7 @@ var (
 	}
 
 	moduleURL         string
-	moduleFlags       = pflag.NewFlagSet("module", pflag.ContinueOnError)
+	moduleDisabled    bool
```

Comment:
"Disabled" can be misleading. For example in shell you can start with -M but should be able load on .cd

Reply:
Renamed to `moduleNoURL`.